package.json

```diff
@@ -33,6 +33,14 @@
     "klaytn sdk",
     "klaytn api"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/klaytn/caver-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/klaytn/caver-js/issues"
+  },
+  "homepage": "https://github.com/klaytn/caver-js#readme",
```

Comment:
The PR for cdn includes the contents of the repository and homepage, and I would like to match the format.

Does homepage have to be linked to readme file?
Should the repository also have git+ in front of url?

Reply:
I think we can change those format. It was just default template when creating a npm package.

Reply:
Okay cool :) then i will comment with suggested change, can you check?